### ISSUE TITLE
Alternative Container Image Solution

### ISSUE THREAD

**Issue by MarkArbogast** [vmware/vmware-aria-operations-integration-sdk#178]
Setting up a registry is a difficult pre-requisite. We need to design a solution that:
1. Will provide an easier path today
2. Will be compatible with the long-term plan that requires changes in VMware Aria Operations

**Comment by MarkArbogast** [comment]
for 1: looking into a script that sets up a registry on a cloud proxy. 
- pull down and run registry image on cloud proxy
- ensure port is opened in cloud proxy firewall

for 2: looking into a combined pak and image deliverable (probably a zip).

**Comment by MarkArbogast** [comment]
For now, let's add more to the requirements section of the README to make it easier for customers to set up their own registry.

1. Fix the link to the "why" anchor
2. Add links for how to set up various container registries (DockerHub, AWS, on-prem, etc.)

**Comment by MarkArbogast** [comment]
Blocked on platform changes. Created separate issue to add registry setup instructions: #225

**Comment by kjrokos** [comment]
Platform changes would be more extensive than we initially thought. We are going forward with better documentation for setting up container registries.
